experimenter/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx

```diff
@@ -170,11 +171,19 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
             <tr>
               <th>Required Experiments</th>
               <td>
-                <ExperimentList experiments={experiment.requiredExperiments} />
+                <ExperimentList
+                  experiments={experiment.requiredExperimentsBranches.map(
+                    (eb) => eb.requiredExperiment,
```

Comment:
Nit:
```suggestion
                    (rb) => rb.requiredExperiment,
```

Reply:
I wouldn't focus on nits of this form, these variable names are so inconsequential and typescript will keep them intact.

Reply:
Ok